lib/index.js

```diff
@@ -0,0 +1,33 @@
+var fs = require('fs');
+var path = require('path');
+
+/**
+ * Copy named files from a directory to a new location
+ *
+ * @param  {Array}  from    Array of files to copy to target directory
+ * @param  {String} destDir directory path to copy files to (no trailing slash)
+ *
+ * @return {String}         Success or failure message
+ */
+exports.copyConfig = function(from, destDir) {
+    if (!Array.isArray(from)) {
+        return 'ERR: Filenames not provided as an array';
+    }
+
+    from.forEach(function(fileName) {
+        try {
+            var destFileName = path.join(destDir, path.basename(fileName))
+
+            console.log('src: ', fileName);
+            console.log('dest: ', destFileName);
+
+            var readFile = fs.readFileSync(fileName);
+
+            fs.writeFileSync(destFileName, readFile);
+        } catch (ex) {
+            return 'ERR: ' + ex.message;
+        }
+    });
+
+    return 'SUCCESS!: ' + from.length + ' file(s) copied successfully';
```

Comment:
Something about console logging a function that console logs itself that which returns 'err' inside forEach that is only skipping that loop and returning success at the end.

I think it'd be best to not have the console log of the output back in fs-default-project-config.js, just call the function, let it manage it's own output, and instead of returning 'err: ', console error.